### ISSUE TITLE
Cannot add filter for `NOT FIND_IN_SET`

### ISSUE THREAD

**Issue by tzyganu** [magento/community-features#133]
Moved here from magento/Magento2#20603: 
As a developer I cannot add a condition to a collection for `NOT FIND_IN_SET` in the regular way with `addAttributeToFilter` or `addFieldToFilter` because `Magento\Framework\DB\Adapter\Pdo\Mysql::prepareSqlCondition` does not contain a map item for NOT FIND_IN_SET.
I always have to do with `Zend_Expr` approach which is not that "pretty".

Possible solution

Add a new map item in the $conditionKeyMap array from the method mentioned above:
```
$conditionKeyMap = [
    ...
   'nfinset' => "NOT FIND_IN_SET(?, {{fieldName}})",
];
```

**Comment by BarryBadpak** [comment]
This has already been added in https://github.com/magento/magento2/commit/734a43f3ac1a684c60e665009cb5aa7f55d29cf3 and released in 2.3.2

**Comment by tzyganu** [comment]
@BarryBadpak 2.5h before I posted this. :+1: Someone can see the future. Nice
